source/optional_features/welsh_language/index.html.md.erb

```diff
@@ -7,23 +7,38 @@ weight: 5220
 
 # Use Welsh on your payment pages
 
-GOV.UK Pay supports Welsh-language payment pages.
+GOV.UK Pay supports Welsh language payment pages.
 
 You can also [create payment links in Welsh](https://www.payments.service.gov.uk/govuk-payment-pages/).
 
-To use Welsh, include `"language": "cy"` in the API request when you [create a new payment](/making_payments/#creating-a-payment). Your users will see Welsh text on the payment pages as they complete the [payment flow](/payment_flow).  If you do not specify a language,
-GOV.UK Pay will default to using English-language payment pages.
+To use Welsh, include `"language": "cy"` in the API request when you [create a new payment](/making_payments/#creating-a-payment).
 
-To add a Welsh service name to your service, sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) and select **Edit name**.
+This example request creates a £50 fine that will use Welsh language payment pages:
 
-If you add a Welsh service name to your service, your users will see this on Welsh-language
-payment pages instead of the English service name.
+`POST /v1/payments`
 
-When your user makes a payment, they will also see a
-[`description`](/payment_flow/#4-your-user-enters-their-payment-details). If you use Welsh-language
-payment pages, you may also want to use Welsh text in the
-`description`.
+```json
+{
+  "amount": 5000,
+  "reference" : "WA5678",
+  "description": "Talu dirwy.",
+  "return_url": "https://your.service.gov.uk/completed",
+  "language": "cy"
+}
+```
 
-GOV.UK Pay does not automatically translate emails into Welsh. If you want to send your own emails in Welsh, stop GOV.UK Pay sending emails by [selecting an account in the GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services) and selecting **Settings**.
+GOV.UK Pay defaults to English language payment pages if you do not send the `language` parameter.
+
+Your users see a [description of the payment](/payment_flow/#4-your-user-enters-their-payment-details) when they pay. If you use Welsh language payment pages, you may also want to use Welsh text in the `description`.
+
+GOV.UK Pay does not automatically translate emails into Welsh. If you want to send your own emails in Welsh, stop GOV.UK Pay sending emails by selecting a service in [the GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services) and selecting **Settings**.
```

Comment:
```suggestion
GOV.UK Pay does not automatically translate emails into Welsh. To stop GOV.UK Pay sending emails in English:

1. Select a service in [the GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services).
1. Select **Settings**.
```

Reply:
I think I may have misunderstood here but it seems like a step by step process - we can review/ run through together once you re back.

Reply:
I agree this should be step by step. I've changed it myself though, rather than accept your suggestion, because there's a few more steps I needed to include. Hope that's okay!